### PR TITLE
Changes in CMakeLists.txt files needed to build a project for Windows…

### PR DESCRIPTION
**Pull request by dkozinski**
… in the MinGW environment using the MSVC compiler

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+# MinGW shell
+#
+# $ cd /home/${USER}/git/xeve
+# $ mkdir build_main
+# $ cd build_main
+# $ cmake .. -G "MinGW Makefiles" -DCMAKE_INSTALL_PREFIX=/usr
+# $ cmake --build . -j <threads>
+# $ cmake --install .
+# $ cmake --build . --package
+
```
Comment (dariusz-f) [inline CMakeLists.txt:47]: In my opinion it will be better to place this guide on project wiki page, not here.

File: CMakeLists.txt [hunk 2]
```diff
@@ -58,6 +102,8 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 # Set compiler flags and options.
 if( MSVC )
+# set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /MTd")
+# set (CMAKE_LIB_LINKER_FLAGS "${CMAKE_LIB_LINKER_FLAGS} /NODEFAULTLIB:libucrt.lib")
 elseif( UNIX OR MINGW )
```
Comment (dariusz-f) [inline CMakeLists.txt:107]: Is this if else is necessary in this situation when windows code is commented out?

**Review by dariusz-f — CHANGES_REQUESTED** [review]

**Review by dariusz-f — APPROVED** [review]
looks good

**Review by kpchoi — APPROVED** [review]
LGTM

**Pull request merged**